projects/fsastorefrontlib/src/cms-components/dynamic-forms/form-components/dynamic-select/dynamic-select.component.ts

```diff
@@ -83,6 +83,5 @@ export class DynamicSelectComponent extends AbstractFormComponent
   assignOptions(options: any[]) {
     this.options$ = of(options);
     this.changeDetectorRef.detectChanges();
-    this.group.get(this.config.name).setValue(null);
```

Comment:
Why is this removed? This should reset values if the fields are mutually dependant. Check auto general information form (vehicle make, model, type, year fields). 
Without it, form will be valid even if customer changes for example vehicle make to something different and other fields (model, type, year) are empty.

Reply:
perhaps this would be a good place to add a comment explaining what this does.